components/feature/accounts-push/src/main/java/mozilla/components/feature/accounts/push/FxaPushSupportFeature.kt

```diff
@@ -203,6 +203,11 @@ internal class ConstellationObserver(
         }
 
         logger.info("We have been notified that our push subscription has expired; re-subscribing.")
+        push.unsubscribe(
+            scope = scope,
+            onUnsubscribeError = { logger.warn("Un-subscribing failed; subscribe may return the same endpoint", it) },
```

Comment:
Can we leave a sentry breadcrumb here? Ideally this would also be a failed assertion in sentry (submit caught exception) for cases that aren't network related.

Reply:
Good point!